src/modules/ce/components/referral/ReferralDetailContent.tsx

```diff
@@ -207,15 +207,17 @@ const ReferralDetailContent: React.FC<Props> = ({ referral }) => {
           <CommonDetailGrid rows={sourceEnrollmentDetails} />
         </CommonCollapsibleCard>
       )}
-      {eligibilityRequirementsDetails && (
```

Comment:
No need to show this if the project has no requirements. For example for a direct referral, this is just an empty collapse header.